src/App/DocumentPyImp.cpp

```diff
@@ -334,7 +345,10 @@ PyObject*  DocumentPy::openTransaction(PyObject *args)
     else if (PyString_Check(value)) {
         cmd = PyString_AsString(value);
     }
-    getDocumentPtr()->openTransaction(cmd.c_str());
+#endif
+    else
+        return NULL;

```

Comment:
Just returning NULL is wrong. You must also set the error state. I will do it qickly